gaphor/tests/testcase.py

```diff
@@ -83,24 +83,23 @@ def allow(self, line, handle, item, port=None):
         adapter = Connector(item, line)
         return adapter.allow(handle, port)
 
-    def connect(self, line, handle, item, port=None):
+    # def connect(self, line, handle, item, port=None):
+    def connect(self, line, handle, item):
         """Connect line's handle to an item.
 
         If port is not provided, then first port is used.
```

Comment:
I think this docstring needs to be updated since port is no longer a parameter.

Reply:
:+1: